### PR TITLE
Adding markdown and imagie for USCIS sensor

### PR DESCRIPTION
**Pull request by meauxt**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13764>

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.uscis.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+comments: false
+sharing: true
+footer: true
+logo: uscis.jpg
+ha_category: Sensor
+ha_release: 0.68
+ha_iot_class: "Cloud Polling"
+---
+
+The `uscis` sensor component allows you to monitor the cost of Washington Dominion Energy.
```
Comment (arsaboo) [inline source/_components/sensor.uscis.markdown]: Incorrect description

File: source/_components/sensor.uscis.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  required: false
+  default: USCIS
+  type: string
+case_id:
+  description: Case Number used to get the case details from USCIS web client.
+  required: true
+  type: string
+{% endconfiguration %}
+
+All the data will be fetch from  [USCIS](https://egov.uscis.gov/casestatus/displayLogon.do) using your Case number.
```
Comment (arsaboo) [inline source/_components/sensor.uscis.markdown]: The link should be https://egov.uscis.gov/casestatus/mycasestatus.do

**Comment by meauxt** [comment]
@arsaboo  changes has been made..

File: source/_components/sensor.uscis.markdown [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+comments: false
+sharing: true
+footer: true
+logo: uscis.jpg
+ha_category: Sensor
+ha_release: 0.68
+ha_iot_class: "Cloud Polling"
+---
+
+The `uscis` sensor component allows you get updates on you USCIS case using your case number/ receipt number
```
Comment (rytilahti) [inline source/_components/sensor.uscis.markdown]: You may want to open what is USCIS a bit in the description, at least by adding a link there.

* "on your USCIS case"
* "case number / receipt number" (whitespace)

File: source/_components/sensor.uscis.markdown [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Example for `configuration.yaml` :
+
+```yaml
+sensor:
+  - platform: uscis
+    case_id: REQUIRED_CASE_NUMBER
```
Comment (rytilahti) [inline source/_components/sensor.uscis.markdown]: Is this a number or a string (as explained below)? IIRC the example should be a real working example one can simply copy & paste to his/her config with minor changes.

**Comment by meauxt** [comment]
@rytilahti changes has been made.

**Review by arsaboo — APPROVED** [review]
Looks good...can be merged when parent PR is merged.

**Pull request merged**